packages/cdk8s/BUILD.bazel

```diff
@@ -0,0 +1,104 @@
+package(default_visibility = ["//visibility:public"])
+
+load("//:packages.bzl", "CONSTRUCTS_PACKAGE_VERSION")
+load("//tools:defaults.bzl", "jest_test", "jsii_library", "jsii_package", "package_json", "ts_library")
+
+MODULE_NAME = "cdk8s"
+
+# Note, we could easily make this into a macro
+JSII_TARGETS = str({
+    "dotnet": {
+        "namespace": "Org.Cdk8s",
+        "packageId": "Org.Cdk8s",
+    },
+    "java": {
+        "maven": {
+            "artifactId": "cdk8s",
+            "groupId": "org.cdk8s",
+        },
+        "package": "org.cdk8s",
+    },
+    "python": {
+        "distName": "cdk8s",
+        "module": "cdk8s",
+    },
+})
+
+package_json(
```

Comment:
I am wondering if we even need `package_json` here. Seems like we can probably merge it (and `jsii_package`) into `jsii_library` which will just do everything. I don't see any reason to split those up really.

Basically, I'd expect this file to look something like this:

```bzl
jsii_library(
    name = "lib",
    desc = "Cloud Development Kit for Kubernetes",
    stability = "experimental",
    jsii_targets = JSII_TARGETS,
    srcs = glob(
        [ "**/*.ts", "README.md" ],
        exclude = ["test/**"],
    ),
    module_name = "cdk8s",
    dev_deps = [
        "@npm//@types/json-stable-stringify",
        "@npm//@types/node",
        "@npm//@types/yaml",
        "@npm//constructs", # this needs to be pinned to 2.0.0
    ],
    peer_deps = [
        "@npm//constructs",  # this needs to be ^2.0.0 (not sure how to express)
    ],
    bundled_deps = [
        "json-stable-stringify",
        "yaml"
    ]
)
```

Something along those lines.

Reply:
I think it's important to have separate, same with `jsii_package`, because it's a separate action. We can create a macro to encapsulate the behavior you're talking about, but leaving these compose-able gives us some advantages. E.g. we don't need to recompile a library every time we want to package it. Or we can reuse the package.json elsewhere without compiling the library (important for test path resolution).